crates/compiler/gen_dev/src/lib.rs

```diff
@@ -1217,6 +1263,226 @@ trait Backend<'a> {
                 let intrinsic = bitcode::NUM_IS_MULTIPLE_OF[int_width].to_string();
                 self.build_fn_call(sym, intrinsic, args, arg_layouts, ret_layout);
             }
+            LowLevel::ListSublist => {
+                //    list: RocList,
+                //    alignment: u32,
+                //    element_width: usize,
+                //    start: usize,
+                //    len: usize,
+                //    dec: Dec,
+
+                let list = args[0];
+                let start = args[1];
+                let len = args[2];
+
+                let list_layout = arg_layouts[0];
+                let element_layout = match self.interner().get(list_layout) {
+                    Layout::Builtin(Builtin::List(e)) => e,
+                    _ => unreachable!(),
+                };
+
+                let (element_width_int, alignment_int) =
+                    self.interner().stack_size_and_alignment(element_layout);
+
+                let alignment = self.debug_symbol("alignment");
+                self.load_literal_i32(&alignment, Ord::max(alignment_int, 8) as i32);
+
+                let element_width = self.debug_symbol("element_width");
+                self.load_literal_i64(&element_width, element_width_int as i64);
```

Comment:
This is repeated a ton, do we want to make it a helper function that return a tuple?